.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+## Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Related Issues:
+- # (issue)
+
+## Type of change
```

Comment:
If you mean type of change could be inferred from the issue, I agree, but having breaking change could not be obvious from the issue alone. Overall it could be useful sometime and causing no harm.

Reply:
No I mean the type of change should be in the issue. Usually via tags but could also be added in the title of the issue. A PR is more a review of a task so that it can be merged. The reviewer should check the issue and make sure all has been done. So that's why I think it makes more sense to put all these points in the issue instead of the PR.

Reply:
Okay. I removed `Type of change` section.